actors/Weapons/Slot3/SSG.dec

```diff
@@ -717,9 +717,9 @@ ACTOR SSG : PB_weapon Replaces SuperShotgun
 	HalfReloadRight: 
         SHT9 EEFFGGHH 1
 	    R6FR OOPP 1
-        TNT1 A 0 A_PlaySound("weapons/ssg/inspect2", 8)
+        TNT1 A 0 A_PlaySound("weapons/ssg/in", 8)
```

Comment:
Going to fix to have the new sounds like the current master branch. I didn't realize that I made a fuck up here by importing possibly older code. Thankfully it doesn't affect much, easy fix.